scripts/store-schema.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import swapiSchema from '../src/schema';
+import { printSchema } from 'graphql/utilities';
+import { writeFileSync } from 'fs'
+
+try {
+  var output = printSchema(swapiSchema);
+  writeFileSync('schema.graphql', output, 'utf8')
+} catch (error) {
+  console.error(error);
+  console.error(error.stack);
+}
```

Comment:
What does this try/catch solve? Does it give you more information about the errors?

Reply:
I just kept it fromthe other script which acts the same, unsure

Reply:
Fair enough; I’m guessing it’s required to see the error metadata like location etc. 🤷‍♂️